app/src/main/java/com/developersbreach/composeactors/data/movie/repository/MovieRepositoryImpl.kt

```diff
@@ -9,87 +9,85 @@ import com.developersbreach.composeactors.data.movie.model.Flatrate
 import com.developersbreach.composeactors.data.movie.model.Movie
 import com.developersbreach.composeactors.data.movie.model.MovieDetail
 import com.developersbreach.composeactors.data.movie.remote.MovieApi
-import java.util.*
-import javax.inject.Inject
-import javax.inject.Singleton
+import java.util.Locale
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
+import javax.inject.Inject
+import javax.inject.Singleton
 
 @Singleton
-class MovieRepositoryImpl @Inject constructor(
-    private val movieApi: MovieApi,
-    private val databaseDataSource: DatabaseDataSource
-) : MovieRepository {
+class MovieRepositoryImpl
+    @Inject
+    constructor(
+        private val movieApi: MovieApi,
+        private val databaseDataSource: DatabaseDataSource,
+    ) : MovieRepository {
+        override suspend fun getNowPlayingMovies(page: Int): Either<Throwable, PagedResponse<Movie>> =
+            withContext(Dispatchers.IO) {
+                movieApi.getNowPlayingMovies(page)
+            }
 
-    override suspend fun getNowPlayingMovies(
-        page: Int
-    ): Either<Throwable, PagedResponse<Movie>> = withContext(Dispatchers.IO) {
-        movieApi.getNowPlayingMovies(page)
-    }
+        override suspend fun getUpcomingMovies(page: Int): Either<Throwable, List<Movie>> =
+            withContext(Dispatchers.IO) {
+                movieApi.getUpcomingMovies(page).map {
+                    it.data
+                }
+            }
 
-    override suspend fun getUpcomingMovies(
-        page: Int
-    ): Either<Throwable, List<Movie>> = withContext(Dispatchers.IO) {
-        movieApi.getUpcomingMovies(page).map {
-            it.data
-        }
-    }
+        override suspend fun getMovieDetails(movieId: Int): Either<Throwable, MovieDetail> =
+            withContext(Dispatchers.IO) {
+                movieApi.getMovieDetails(movieId)
+            }
 
-    override suspend fun getMovieDetails(
-        movieId: Int
-    ): Either<Throwable, MovieDetail> = withContext(Dispatchers.IO) {
-        movieApi.getMovieDetails(movieId)
-    }
+        override suspend fun getSimilarMovies(
+            movieId: Int,
+            page: Int,
+        ): Either<Throwable, List<Movie>> =
+            withContext(Dispatchers.IO) {
```

Comment:
Need to remove the multiline expression formatting and can be placed in same line.
```kotlin
ktlint_standard_multiline-expression-wrapping = disabled
```